deployments/optimism/usdc/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+  "baseTokenPriceFeed": "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+  "pauseGuardian": "0x3fFd6c073a4ba24a113B18C8F373569640916A45",
```

Comment:
Just to double check, was multisig created from the [`clone-multisig`](https://github.com/compound-finance/comet/tree/main?tab=readme-ov-file#clone-multisig) script?

Reply:
Yes

Here the transaction that initiated the creation - https://optimistic.etherscan.io/tx/0x1be034f38200ea3d45e81c4b529ec7a782f26e725ff83edc9b75f46f6ed3891e

Here is the result - https://optimistic.etherscan.io/address/0x3fFd6c073a4ba24a113B18C8F373569640916A45#internaltx